components/IngredientDetails.js

```diff
@@ -12,13 +12,16 @@ import {
   CardFooter,
   DetailsCard,
   DeleteButton,
-  ConfirmationDialog,
+  ConfirmDialog,
+  Button,
+  EditButton,
+  Overlay,
```

Comment:
```suggestion
  Overlay,
```
The naming for 'Overlay' is for me not really clear, maybe there is a clearer naming what it's doing?